database/migrations/2024_09_11_200845_create_server_credentials_table.php

```diff
@@ -15,6 +15,7 @@ public function up(): void
         Schema::create('server_credentials', function (Blueprint $table) {
             $table->id();
             $table->foreignIdFor(Tenant::class);
+            $table->string('title');
```

Comment:
please don't do this. we have to assume that people have already run existing migrations on their project and they will "upgrade" with this PR. Thus, we need a new migration

Reply:
ugh fine